.github/workflows/changelog.yml

```
@@ -0,0 +1,119 @@
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: feat/v7-develop
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Determine entry type
+        id: entry-type
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)